castle/cms/browser/imageinfo.py

```diff
@@ -0,0 +1,14 @@
+import json
+from Products.Five.browser import BrowserView
+from castle.cms import utils
+
+class ImageInfoView(BrowserView):
+
+    def __init__(self, context, request):
```

Comment:
Not very important, but for future reference, this declaration isn't necessary. You should only need to override __init__ if you're doing something non-standard

Reply:
👌